src/components/footer/_macro.spec.js

```diff
@@ -478,6 +478,22 @@ describe('macro: footer', () => {
           expect($('.custom-logo').length).toBe(1);
         });
       });
+
+      it('it has the additional logo image and link', () => {
+        const $ = cheerio.load(
+          renderComponent('footer', {
+            legal: EXAMPLE_LEGAL_PARAM,
+            crest: true,
+            logo: {
+              logoURL: '#0',
+              logoImage: '<img src="a-logo.svg" class="additional-logo">',
+            },
+          }),
+        );
+
+        expect($('.ons-footer__additional-logo').attr('href')).toBe('#0');
+        expect($('.additional-logo').attr('src')).toBe('a-logo.svg');
```

Comment:
Because you are adding the "additional-logo" class here, this line doesn't actually test what you think it does, its testing that there is an element with that class which has the image "a-logo.svg". But you have supplied all of that yourself in the test above

Reply:
This tests that the footer has the 'a-logo.svg' image. It doesn't matter if we add the class in the test or not. If the footer doesn't have the image, the test will still fail

Reply:
I see what you are saying, but its just testing that the macro puts that html somewhere on the page. Which is ok, I just thought there might be a better way of testing it. Like if you were able to select the element using classes that were set in the macro programatically and without adding the class in the test like you have done, that way you would be able to tell that it was the correct element that was getting the image. Have a look at the css selectors in the masthead logo tests

Reply:
I agree with you, but didn't want to add a class which is not doing anything other than being used for testing.

Reply:
You don't necessarily need to use a class on that element, you can do quite a lot with css selectors

Reply:
This issue hasn't been addressed